modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties

```diff
@@ -80,6 +80,12 @@
     #
     #liferay.workspace.modules.jsp.precompile.enabled=false
 
+    #
+    # Set this property to build Node.js-style projects using Yarn. The default
+    # value is npm.
+    #
+    #liferay.workspace.node.package.manager=yarn
```

Comment:
hey @lawrence-lee  is this the default?

Reply:
npm is default. Thats why i worded the blurb the way i did. I thought it was the shortest most concise way to saying there are 2 options `npm` and `yarn`